### PR TITLE
Config table

### PR DESCRIPTION
**Pull request by jayajoseph1**
changes made in postmapping

### CODE REVIEW

File: src/main/java/com/ust/sourcecourse/configuration/repository/SourceTableRepository.java [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.ust.sourcecourse.configuration.entity.SourceTable;
+import com.ust.sourcecourse.configuration.request.SourceTableRequest;
+
+public interface SourceTableRepository extends JpaRepository<SourceTable, Long> {
+
+	SourceTable save(SourceTableRequest tableRequest);
+
+	List<SourceTable> findByUidIn(List<Long> sourceTableUids);
```
[the remark this answers is not in the document]
Comment (jayajoseph1) [reply src/main/java/com/ust/sourcecourse/configuration/repository/SourceTableRepository.java]: resolved

File: src/main/java/com/ust/sourcecourse/configuration/response/ProjectTableResponse.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ProjectTableResponse {
+
+	private Long uid;
+
+	private ProjectResponse project;
```
[the remark this answers is not in the document]
Comment (jayajoseph1) [reply src/main/java/com/ust/sourcecourse/configuration/response/ProjectTableResponse.java]: resolved

File: src/main/java/com/ust/sourcecourse/configuration/controller/ProjectTableController.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+	@PostMapping
+	public ResponseEntity<List<ProjectTableResponse>> createProjectTable(
+			@Valid @RequestBody ProjectTableRequest ProjTableReq) {
+		List<ProjectTableResponse>projectTableResponse = projectTableService.createProjectTable(ProjTableReq);
+		return ResponseEntity.status(HttpStatus.CREATED.value()).body(projectTableResponse);
+	}
+
+	@GetMapping("/{id}")
+	public ResponseEntity<Optional<ProjectTable>> getProjectTable(@PathVariable Long id) {
```
[the remark this answers is not in the document]
Comment (jayajoseph1) [reply src/main/java/com/ust/sourcecourse/configuration/controller/ProjectTableController.java]: resolved

[the remark this answers is not in the document]
Comment (jayajoseph1) [reply src/main/java/com/ust/sourcecourse/configuration/controller/ProjectTableController.java]: resolved

File: src/main/java/com/ust/sourcecourse/configuration/controller/ProjectTableController.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+		Optional<ProjectTable> sourceTable = projectTableService.getProjectTable(id);
+		return ResponseEntity.ok(sourceTable);
+	}
+
+	@PutMapping("/{id}")
+	public ResponseEntity<Optional<ProjectTable>> updateProjectTable(@PathVariable Long id,
```
[the remark this answers is not in the document]
Comment (jayajoseph1) [reply src/main/java/com/ust/sourcecourse/configuration/controller/ProjectTableController.java]: resolved

**Comment by jayajoseph1** [comment]
changes resolved

**Review by shijukbabu-ust — APPROVED** [review]

**Pull request merged**